pybedlite/overlap_detector.py

```diff
@@ -55,6 +56,24 @@
 from pybedlite.bed_source import BedSource
 from pybedlite.bed_record import BedRecord
 
+UCSC_STRAND_REGEX = re.compile(r".*\((\+|-)\)$")
+"""
+Match a parenthetically enclosed strand at the end of a position-formatted interval.
+
+Groups:
+    1: the strand ("+" or "-")
+"""
+
+UCSC_INTERVAL_REGEX = re.compile(r"^(.*):(\d+)-(\d+)$")
+"""
+Match a position-formatted interval.
+
+Groups:
+    1: the refname (or chromosome)
+    2: the 1-based start position
+    3: the 1-based closed end position
+"""
```

Comment:
This ok? I don't think `Groups:` is an established section the way `Attributes:` is but thought it helpful to document